index.html

```diff
@@ -19,15 +19,20 @@
   </head>
 
   <body>
+    <div class="orders-cont">
+      <button class="x-button"><i class="fa-solid fa-xmark"></i></button>
+      <ul class="orders">
+        <li class="linkers"><a class="nav-link" href="#cards"> Portoflio</a></li> 
+        <li class="linkers"><a class="nav-link" href="#desk">About</a></li>
+        <li class="linkers"><a class="nav-link" href="#the-final">Contact</a></li>
+      </ul>
+    </div>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure that the navigation list items are visible on the Desktop so they can be clicked and redirect the user to their expected sections. Please make sure to fix this issue to make your portfolio complete and look more professional. I know you can do it :muscle: :muscle: